src/qt/bitcoingui.cpp

```diff
@@ -713,13 +713,19 @@ void BitcoinGUI::updateNetworkState()
     default: icon = ":/icons/connect_4"; break;
     }
 
+    QString tooltip;
+
     if (clientModel->getNetworkActive()) {
-        connectionsControl->setToolTip(tr("%n active connection(s) to Bitcoin network", "", count));
+        tooltip = tr("%n active connection(s) to Bitcoin network", "", count) + QString(".<br>") + tr("Click to disable network activity.");
     } else {
-        connectionsControl->setToolTip(tr("Network activity disabled"));
+        tooltip = tr("Network activity disabled") + QString(".<br>") + tr("Click to enable network activity again.");
```

Comment:
I believe this is a new string, so prefer to put the period in the translation. (keep the <br> outside however)